### PR TITLE
feat(compose): Auto-configure Keycloak in Docker Compose for UI

### PR DESCRIPTION
**Pull request by mmurto**
While in longer term this should probably be done with Terraform/Pulumi, configure it with the JSON to ease development.

### CODE REVIEW

**Comment by mmurto** [comment]
Only real changes are the client configuration and adding admin to the SUPERUSER group. The file was generated as instructed below, which also added some other changes.

https://github.com/eclipse-apoapsis/ort-server/blob/88204b7a1a58eb88494bbe8edaa8f219bed3a60a/scripts/docker/keycloak/init-keycloak.sh#L17-L23

File: scripts/docker/keycloak/master-realm.json [hunk 1]
```diff
@@ -419,7 +420,7 @@
     "requiredActions" : [ ],
     "realmRoles" : [ "default-roles-master", "admin" ],
     "notBefore" : 0,
-    "groups" : [ ]
+    "groups" : [ "/SUPERUSERS" ]
```
Comment (mnonnenmacher) [inline scripts/docker/keycloak/master-realm.json:423]: Is this required? There is already the ort-admin user which is in the superuser group and which is supposed to be used for testing. On the other hand, I guess it doesn't hurt if admin in this group as well.

Comment (mmurto) [reply scripts/docker/keycloak/master-realm.json:423]: It's a little bit less friction via the UI, Keycloak and the UI share the session. If you want to change something in Keycloak and see its effect in the UI, you need to change the users in between if using the same browser.

So not required, but if there are no strong preferences from you, I'd like to keep it.

Comment (mnonnenmacher) [reply scripts/docker/keycloak/master-realm.json:423]: Thanks, that makes sense.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**